src/SFA.DAS.PSRService.Web/Views/Report/AlreadySubmitted.cshtml

```diff
@@ -1,10 +1,10 @@
 ﻿@model object
 
 @{
-
-    ViewBag.Title = $"Current report already submitted";
-
+    ViewBag.Title = "Current report already submitted";
+    ViewBag.GaData.Vpv = "/public-sector-reporting/report/page-already-submitted";
     Layout = "_Layout";
+    ViewBag.ZenDeskLabel = "reporting-report-submitted";
```

Comment:
If you go with your workaround then all these ViewBag settings do nothing and potentially mislead so should be removing them

Reply:
👍